From a7a4f1446231a8e9105399af258ae8b047ddb125 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ale=C5=A1=20Mat=C4=9Bj?= <amatej@redhat.com>
Date: Tue, 13 Jan 2026 16:27:41 +0100
Subject: [PATCH 4/8] ruby bindings: ignore unused-but-set-variable warning
 (error)

Swig generated code produces unused-but-set-variable wanrings.
We don't control that so just ignore it.
---
 bindings/ruby/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bindings/ruby/CMakeLists.txt b/bindings/ruby/CMakeLists.txt
index 53248396..cae42e26 100644
--- a/bindings/ruby/CMakeLists.txt
+++ b/bindings/ruby/CMakeLists.txt
@@ -15,6 +15,7 @@ include_directories(${Ruby_INCLUDE_DIRS})
 if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     add_compile_options(-Wno-attribute-warning)
 endif()
+add_compile_options(-Wno-unused-but-set-variable)
 
 function(add_ruby_module LIBRARY_NAME MODULE_NAME)
     set(TARGET_NAME "ruby_${MODULE_NAME}")
-- 
2.52.0

